README.md

```diff
@@ -615,12 +613,120 @@ This will start generating a new up-to-date database under `/tmp/madara_new`.
 Once this process is over, the warp update sender node will automatically
 shutdown while the warp update receiver will take its place.
 
-> [!WARNING]
-> As of now, the warp update receiver has its rpc disabled, even after the
-> migration process has completed. This will be fixed in the future, so that
-> services that would otherwise conflict with the sender node will automatically
-> start after the migration has finished, allowing for migrations with 0
-> downtime.
+> [!NOTE]
+> You might already have noticed this line which appears at the end of the sync:
+> `📱 Running JSON-RPC server at 127.0.0.1:9944 ...`. More about this in the
+> next section
+
+### Achieving Zero Downtime
+
+Suppose your are an RPC service provider and your node is also running an RPC
+server and exposing it to your clients: if you have to shut it down or restart
+it for the duration of a migration this will result in downtime for your service
+and added complexity in setting up redundancies.
```

Comment:
Yea fair, though this is still a bit faster than that due to being on a local network. It would also be quite easy to make this a lot faster "in the future :tm: " by computing the state root once the migration has completed, instead of at each block which is what we do on a normal sync. Something to keep in mind.